### PR TITLE
Update observation time

### PR DESCRIPTION
**Pull request by cquiroz**
Adds a new mutation to update the observation time, this could be used by the scheduler to set the expected execution time and eventually the duration

### CODE REVIEW

File: modules/service/src/test/scala/lucuma/odb/graphql/mutation/updateObservationsTimes.scala [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        }
+      }
+    """.asRight
+
+    val expected1 = json"""
+      {
+        "updateObservationsTimes": {
+          "observations": [
+            {
+              "observationTime": null
```
Comment (swalker2m) [inline modules/service/src/test/scala/lucuma/odb/graphql/mutation/updateObservationsTimes.scala:124]: It occurred to me that setting the observation time to the same value for multiple observations might be something that is never really done, but it does seem handy to be able to wipe them all out in one mutation like this.

Comment (cquiroz) [reply modules/service/src/test/scala/lucuma/odb/graphql/mutation/updateObservationsTimes.scala:124]: That's right. it tests the machinery of multiple observations though

File: modules/service/src/main/resources/db/migration/V0904__observation_time.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+-- Rename visualization time
+ALTER TABLE t_observation
+RENAME COLUMN c_visualization_time TO c_observation_time;
+
+-- Re-create v_observation to include the science band
```
Comment (toddburnside) [inline modules/service/src/main/resources/db/migration/V0904__observation_time.sql:5]: I don't see the science band in the view...

**Review by toddburnside — APPROVED** [review]
LGTM

**Pull request merged**